packages/fetchye/src/makeServerFetchye.js

```diff
@@ -27,18 +26,27 @@ const makeServerFetchye = ({
   fetchClient,
 }) => async (
   key,
-  { mapOptionsToKey = (options) => options, ...options } = { },
+  options = {},
   fetcher = ssrFetcher
 ) => {
   const { cacheSelector } = cache;
-  const computedKey = computeKey(key, defaultMapOptionsToKey(mapOptionsToKey(options)));
+  const computedKey = computeKey(key, options);
+  const run = () => runAsync({
+    dispatch,
+    computedKey,
```

Comment:
```suggestion
    computedKey: computeKey(key, options),
```
Same as in useFetchye:

the dynamic headers need to be recreated inside `run`